pkg/registry/core/pod/strategy.go

```diff
@@ -19,6 +19,9 @@ package pod
 import (
 	"context"
 	"fmt"
+	"k8s.io/klog/v2"
+	"k8s.io/kubectl/pkg/cmd/util/podcmd"
+	apiscorev1 "k8s.io/kubernetes/pkg/apis/core/v1"
```

Comment:
These imports should not go here; please fix the imports formatting

Reply:
Sure, fixed.